engine/src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -7,16 +7,20 @@ import cromwell.engine.ExecutionStatus
 import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.db.ExecutionDatabaseKey
 import cromwell.engine.workflow.BackendCallKey
+import slick.dbio.Effect.Read
+import slick.jdbc.GetResult
 import slick.profile.RelationalProfile.ColumnOption.Default
 import wdl4s.{NamespaceWithWorkflow, Scatter}
 
 import scala.language.postfixOps
-import scala.util.{Success, Failure, Try}
+import scala.util.{Failure, Success, Try}
 
 case class CallCacheHit(workflowId: String, callName: String)
 case class ExecutionWithCacheData(execution: Execution, cacheHit: Option[CallCacheHit])
 
-case class Execution(workflowExecutionId: Int,
+/** NOTE: the attributes should be in column-order so that executionsWithCacheHitWorkflowAndCall can work */
+case class Execution(executionId: Option[Int] = None,
```

Comment:
you don't need to change the order here, you just need the column order in your select statement to match your constructor order.